packages/api/controllers/hackathon_team_controller.go

```diff
@@ -202,6 +202,27 @@ func DeleteHackathonTeams(c *fiber.Ctx) error {
 
 	key_from_hex, _ := primitive.ObjectIDFromHex(hackathon_team_key)
 
+	var team models.Team
+	err := hackathonTeamCollection.FindOne(ctx, bson.M{"_id": key_from_hex}).Decode(&team)
+
+	if err != nil {
+		return c.Status(http.StatusInternalServerError).JSON(responses.MemberResponse{Status: http.StatusNotFound, Message: "error", Data: &fiber.Map{"data": err.Error()}})
+	}
+
+
+	teamMembers := make([]models.TeamMember, len(team.TeamMembers))
+
+	var tempTeamMember models.TeamMember
+
+	for i := 0; i < len(team.TeamMembers); i++ {
+		key_from_hex, _ := primitive.ObjectIDFromHex(team.TeamMembers[i])
+		_ = teamMembersCollection.FindOne(ctx, bson.M{"_id": key_from_hex}).Decode(&tempTeamMember)
+		teamMembers[i] = tempTeamMember
+	}
+
+	BatchAddTeamMembersToNoParticipants(ctx, teamMembers)
```

Comment:
Can you handle exceptions here?

Reply:
Should it return 500 as status code or?

Reply:
yep

```go
_, err := BatchAddTeamMembersToNoParticipants(ctx, teamMembers)
if err != nil {
    return c.Status(blah, blah, blah, 500, message: "Failed moving members")
}
 
```

Reply:
Yeah I did that, just needed the green light to commit

Reply:
don't be afraid to commit and push 😄. We can easily revert back if something is not ok. By pushing I get notified as a codeowner and I can check the code asap if all is good.